MLH-Fellowship/autoflow
# Create python-package.yml and Add Testing

## Changes made:
#### - Tasks solved
 - New workflow for Github Actions that tests if CLI application works for multiple python versions
 - Added testing for the basic commands currently in the main branch
 - Removed Python Application workflow

## How to test:

Switch into the testing branch in the Autoflow repo. Go to Github Actions and go to the Python Package Workflow under All Workflows on the left side. 

Click the "Create python-package.yml" workflow run at the top of the filtered list. Click "Re-run jobs" to check that the tests work for the current commands.

Feel free to check out the workflow file and test file and to check that the other files have not been changed. 

## Screenshots

<img width="912" alt="Screen Shot 2020-11-20 at 5 14 54 AM" src="https://user-images.githubusercontent.com/51776663/99804243-8c0acb00-2aef-11eb-871c-89a3b9503b35.png">
<img width="1439" alt="Screen Shot 2020-11-20 at 5 15 05 AM" src="https://user-images.githubusercontent.com/51776663/99804247-8dd48e80-2aef-11eb-93a3-cba1c9884bcf.png">
<img width="1438" alt="Screen Shot 2020-11-20 at 5 15 19 AM" src="https://user-images.githubusercontent.com/51776663/99804250-8f05bb80-2aef-11eb-9fd4-10b397e06d68.png">
<img width="1097" alt="Screen Shot 2020-11-20 at 5 15 35 AM" src="https://user-images.githubusercontent.com/51776663/99804256-90cf7f00-2aef-11eb-82d8-e7816fb77755.png">

## Pull Request checklist:

- [x] Check that code compiles.
- [x] Attach screenshots of expected behavior.
- [x] Check that there are no confidential files like `.env` included.
- [x] The changes pass tests locally.
- [x] The changes don't break existing features.


## Pull Request type:

#### - New feature
